packages/edit-post/src/components/sidebar/plugin-document-setting-panel/index.js

```diff
@@ -10,14 +10,19 @@ import { compose } from '@wordpress/compose';
 import { withPluginContext } from '@wordpress/plugins';
 import { withDispatch, withSelect } from '@wordpress/data';
 
+/**
+ * External dependencies
+ */
```

Comment:
Per documented convention, can you place this grouping _before_ the "WordPress dependencies" ?

https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#imports

Reply:
Done.